qr-code/node/web/middleware/auth.js

```diff
@@ -75,12 +29,12 @@ export default function applyAuthMiddleware(app) {
         }
       });
 
+      const redirectUrl = Shopify.Context.IS_EMBEDDED_APP
+        ? Shopify.Utils.getEmbeddedAppUrl(req)
+        : `/?shop=${session.shop}&host=${encodeURIComponent(req.query.host)}`;
```

Comment:
Shouldn't we still be sanitizing the host here?

Reply:
Yeah, good call.  Updated 👍